### PR TITLE
TS: Clean up.

### PR DESCRIPTION
**Pull request by linbingquan**

### CODE REVIEW

File: src/math/Box2.d.ts [hunk 1]
```diff
@@ -16,16 +16,16 @@ export class Box2 {
 	copy( box: Box2 ): this;
 	makeEmpty(): Box2;
 	isEmpty(): boolean;
-	getCenter( target: Vector2 ): Vector2;
-	getSize( target: Vector2 ): Vector2;
+	getCenter( target?: Vector2 ): Vector2;
+	getSize( target?: Vector2 ): Vector2;
```
Comment (Mugen87) [inline src/math/Box2.d.ts:20]: I'm afraid these changes are wrong. `target` is mandatory. Checks like:
```js
if ( target === undefined ) {

	console.warn( 'THREE.Box3: .getSize() target is now required' );
	target = new Vector3();

}
```
are for backwards compatibility only.

Comment (linbingquan) [reply src/math/Box2.d.ts:20]: En... May be this `Vector3.d.ts cross()` method should be corrected

https://github.com/mrdoob/three.js/blob/b4473c25816df4a09405c7d887d5c418ef47ee76/src/math/Vector3.d.ts#L214

https://github.com/mrdoob/three.js/blob/b4473c25816df4a09405c7d887d5c418ef47ee76/src/math/Vector3.js#L513-L524

Comment (Mugen87) [reply src/math/Box2.d.ts:20]: Yes, `w?: Vector3` should be removed.

Comment (linbingquan) [reply src/math/Box2.d.ts]: > Yes, `w?: Vector3` should be removed.

OK.

**Review by Mugen87 — APPROVED** [review]

File: src/math/Vector3.d.ts [hunk 2]
```diff
@@ -211,7 +211,7 @@ export class Vector3 implements Vector {
 	/**
 	 * Sets this vector to cross product of itself and v.
 	 */
-	cross( a: Vector3, w?: Vector3 ): this;
+	cross( a: Vector3, w: Vector3 ): this;
```
Comment (Mugen87) [inline src/math/Vector3.d.ts:214]: The idea was to remove `w: Vector3`^^.

Comment (linbingquan) [reply src/math/Vector3.d.ts:214]: > The idea was to remove `w: Vector3`^^. You can do this for `add()`, too.

I think  `Vector3.d.ts#fromBufferAttribute()` can be removed the `offset`, it makes sense.

**Comment by linbingquan** [comment]
@Mugen87 Can you please to review the new code. I worry about I made a mistake.

File: src/math/Vector3.d.ts [hunk 3]
```diff
@@ -94,7 +94,7 @@ export class Vector3 implements Vector {
 	 */
 	subVectors( a: Vector3, b: Vector3 ): this;
 
-	multiply( v: Vector3 ): this;
+	multiply( v: Vector3, w: Vector3 ): this;
```
Comment (Mugen87) [inline src/math/Vector3.d.ts:97]: It should be only `v: Vector3`.

Comment (linbingquan) [reply src/math/Vector3.d.ts]: > It should be only `v: Vector3`.

Thanks!

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**